react/styles.global.css

```diff
@@ -41,4 +41,28 @@ ul.status {
 
 div.timeline:last-of-type ul.last-of {
   background-image: none;
-} 
\ No newline at end of file
+}
+
+.return-itemsList-body tr {
+  border-bottom: 1px solid #e3e4e6;
+}
+
+.return-itemsList-body tr:last-child {
+  border-bottom: none;
+}
+
+.createReturnRequestContainer .vtex-pageHeader__container {
```

Comment:
Nitpicking here. Should we use kebab-case here?

Reply:
Added